review/.mockery.yaml

```diff
@@ -0,0 +1,11 @@
+with-expecter: True
```

Comment:
I used mockery for generating mocks for testing purposes. Previously I used flags for cli command configuration, but newer versions require config with yaml.